### PR TITLE
add county filter on farms endpoint

### PR DESCRIPTION
**Pull request by Omarabdul3ziz**
### Description
add county filter on farms endpoint

### Changes
- add db filter on farms table
- add tests for the new filter
- fix countries slice appending on nodes aggregation test
- update swagger docs

### Related Issues

- https://github.com/threefoldtech/tfgrid-sdk-go/issues/329

### Checklist

- [x] Tests included
- [x] Build pass
- [ ] Documentation
- [ ] Code format and docstring

### CODE REVIEW

File: grid-proxy/tests/queries/farm_test.go [hunk 1]
```diff
@@ -41,6 +41,11 @@ var farmFilterRandomValueGenerator = map[string]func(agg FarmsAggregate) interfa
 		name := changeCase(agg.farmNames[rand.Intn(len(agg.farmNames))])
 		return &name
 	},
+	"Country": func(_ FarmsAggregate) interface{} {
```
Comment (AbdelrahmanElawady) [inline grid-proxy/tests/queries/farm_test.go:44]: why not like other fields with FarmsAggregate

**Review by AbdelrahmanElawady — APPROVED** [review]
Looks good just a question

File: grid-proxy/tests/queries/farm_test.go (hunk 1 above)
Comment (Omarabdul3ziz) [reply grid-proxy/tests/queries/farm_test.go:44]: the `agg` variable is not intended to be used inside this function. however, it is still needed to satisfy the function signature. I set it as blank to bypass the warnings

Comment (AbdelrahmanElawady) [reply grid-proxy/tests/queries/farm_test.go:44]: I am just wondering why is it not intended to be used inside the function.

Comment (Omarabdul3ziz) [reply grid-proxy/tests/queries/farm_test.go:44]: the farm table itself doesn't have a country field so we got the possible counties from the node aggregation

**Pull request merged**